core/scripts/bareos-config-lib.sh.in

```diff
@@ -762,7 +762,12 @@ get_databases_installed()
 
 get_database_driver_default()
 {
-    DBDRIVER=`get_database_driver`
+    if [ -n "$DBTYPE" ]; then
+      DBDRIVER=$DBTYPE;
+    else
+      DBDRIVER=`get_database_driver`
```

Comment:
```suggestion
      DBDRIVER=$(get_database_driver)
```

Reply:
These Backticks are inherited and it does not make sense to only change parts of the script.